app/Jobs/EsDocument.php

```diff
@@ -66,5 +68,18 @@ public function handle(): void
         $keyName = $model->getKeyName();
         $model->setAttribute($keyName, $id);
         $model->esDeleteDocument();
+        static::incrementStat('delete', get_class($model));
+    }
+
+    private static function incrementStat(string $action, string $class)
```

Comment:
static functions should be above constructor shouldn't they? And set a return type